src/main/java/uk/gov/hmcts/darts/cases/mapper/HearingEntityToCaseHearing.java

```diff
@@ -38,7 +39,11 @@ private Hearing mapToHearing(HearingEntity entity) {
         var transcripts = entity.getTranscriptions()
             .stream()
             .filter(transcriptionEntity -> BooleanUtils.isTrue(transcriptionEntity.getIsManualTranscription())
-                || StringUtils.isNotBlank(transcriptionEntity.getLegacyObjectId()))
+                || StringUtils.isNotBlank(transcriptionEntity.getLegacyObjectId())
+            )
+            .filter(transcriptionEntity -> transcriptionEntity.getTranscriptionDocumentEntities().isEmpty()
+                || transcriptionEntity.getTranscriptionDocumentEntities().stream().noneMatch(TranscriptionDocumentEntity::isHidden)
```

Comment:
Do we want to exclude the entire transcription or just remove the hidden transcription document from the response?

Reply:
The response includes a count of transcriptions. We want to exclude any transcriptions which have any hidden documents associated with them.